Tests/UnitTests/FileShareService.DesktopClientTests/Modules/Admin/AdminViewModelTests.cs

```diff
@@ -19,6 +20,8 @@ public class AdminViewModelTests
         private IKeyValueStore fakeKeyValueStore = null!;
         private IJobsParser fakeJobsParser = null!;
         private IEnvironmentsManager fakeEnvironmentsManager = null!;
+        private readonly ILogger<AdminViewModel> logger;
+        private readonly ILogger<NewBatchJobViewModel> Nlogger;
```

Comment:
Again, please initialise the loggers to A.Fakes

Reply:
Resolved